internal/provider/data_source_gitlab_release_links.go

```diff
@@ -0,0 +1,85 @@
+package provider
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/xanzy/go-gitlab"
+)
+
+var _ = registerDataSource("gitlab_release_links", func() *schema.Resource {
+
+	return &schema.Resource{
+		Description: `The ` + "`gitlab_release_links`" + ` data source allows get details of release links.
+
+**Upstream API**: [GitLab REST API docs](https://docs.gitlab.com/ee/api/releases/links.html)`,
+
+		ReadContext: dataSourceGitlabReleaseLinksRead,
+		Schema: map[string]*schema.Schema{
+			"project": {
+				Description: "The ID or [URL-encoded path of the project](https://docs.gitlab.com/ee/api/index.html#namespaced-path-encoding).",
```

Comment:
While the API "only" accepts URL-encoded paths, that's not the case for this provider, because `go-gitlab` will correctly URL-encode it for you. Let's use this instead:

```suggestion
				Description: "The ID or full path to the project.",
```

Reply:
Done